### DISCUSSION TITLE
About header customization

### DISCUSSION THREAD

**Discussion by Sandunjayasekar** [squidfunk/mkdocs-material#5209] [Q&A] [answered]
![Screenshot (36)](https://user-images.githubusercontent.com/73893725/225882270-6b1725a2-e3f6-41a9-a7f8-4a46852f408d.png)

I want to customize header and add some dropdown menus and more things to header section, But i couldn't find html files for customize it, i tried with include header and attach header file, But it's not worked for me, Can you give me some suggestion and referrence or something about that ?

**Comment by squidfunk** [comment, marked as answer]
Duplicate of #5117 – your own issue.
